### ISSUE TITLE
Missing focus outline on `.btn` components

### ISSUE THREAD

**Issue by hyunbinseo** [hyunbinseo/svelte-kit-template#19]

**Comment by hyunbinseo** [comment]
Fixed in 78f010cf5fff1298fdef5b715bcacfd46d657389
